docs/tutorials/03-on-build.md

```diff
@@ -0,0 +1,103 @@
+![buildah logo](../../logos/buildah-logo_large.png)
+
+# Buildah Tutorial 3 
+## Using ONBUILD in Buildah 
+
+The purpose of this tutorial is to demonstrate how Buildah can use a Dockerfile with the OnBuild command within it or how it can be used with the `buildah config` command.  The ONBUILD command stores a command in the meta data of the image which is then invoked whn the container is created.
+
+A note about this container images compliant with the [Open Container Initiative](https://www.opencontainers.org/) (OCI) [image specification](https://github.com/opencontainers/image-spec) do not support the ONBUILD instruction.  Therefore only containers that use the Docker format can use this technique.  Buildah is capable of working seamlessly with either OCI or Docker compliant images and containers. 
```

Comment:
Add a line to state that buildah defaults to the OCI Format but can be overriden by using the --format docker or by setting the BUILDAH_FORMAT environment variable.

Reply:
I touched this last paragraph up a bit and added the above info, good thought.